File: prognosis-frontend/src/Service/Service.ts

export type Service = {
  serviceId: string;
  name: string;
};

/*
\( NEWID\(\), @GoogleGUID, ([0-9]+), '([0-9]+-[0-9]+-[0-9]+ 02:00:00)'\)
{ id: "00000-00000-00000-00000-00000", serviceId: "00000-00000-00000-00000-00001", count: $1, timestamp: $2, }
*/

File: prognosis-backend/Models/Service.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace prognosis_backend.models;

[Table("service")]
public class Service
{
    [Key]
    [Column("service_id")]
    public Guid ServiceId { get; set; }
    [Column("name")]
    public required string Name { get; set; }
}
